SofaKernel/modules/SofaHelper/src/sofa/helper/fixed_array.h

```diff
@@ -278,7 +278,7 @@ class fixed_array
     }
 
     // size is constant
-    static size_type size()
+    static constexpr size_type size()
```

Comment:
if this is a constexpr is the static usefull anymore ?

Reply:
I let the static because I dont know if people was not using Quater<Real>::size() (I guess not but still)
If I remove the static, people can just to <instanciation>.size()